### DISCUSSION TITLE
imgkit - OSError:  No wkhtmltoimage executable found: "command not found" If this file exists please check that this process can read it. Otherwise please install wkhtmltopdf

### DISCUSSION THREAD

**Discussion by chienhsiang-hung** [vercel/vercel#8268] [Help]
I'm using imgkit in one of my server though install imgkit and wkhtmltopdf is successful. there is an error while using it:

OSError:  No wkhtmltoimage executable found: "command not found" If this file exists please check that this process can read it. Otherwise please install wkhtmltopdf

If we need to specify the path, where is the path to executable file of wkhtmltopdf / wkhtmltoimage 

Thank you

**Comment by amyegan** [comment]
Thanks for reaching out, @chienhsiang-hung. Is this error happening on local, or is it only happening on Vercel?

**Reply by chienhsiang-hung** [reply]
Good morning @amyegan. It's on Vercel.
This is my [test code](https://github.com/chienhsiang-hung/kaggle-profile-summary-card/blob/main/api/htmltopngtest.py) and [dependencies](https://github.com/chienhsiang-hung/kaggle-profile-summary-card/blob/main/requirements.txt) and [test API](https://kaggle-card.chienhsiang-hung.eu.org/api/htmltopngtest?chienhsianghung):

error code for `img = imgkit.from_string(html, False, options=options)`:
```
OSError: No wkhtmltoimage executable found: "command not found" If this file exists please check that this process can read it. Otherwise please install wkhtmltopdf
```

Thank you.

**Reply by amyegan** [reply]
Thanks for sharing this. Does it work if you install the package as part of your [Install Command](https://vercel.com/changelog/dependencies-can-now-be-installed-with-a-custom-command) to make it available before the Build Step?

![image](https://user-images.githubusercontent.com/8247105/181845332-36b3551c-89da-4a77-b16f-fcc313c6dbae.png)

**Reply by chienhsiang-hung** [reply]
> Thanks for sharing this. Does it work if you install the package as part of your [Install Command](https://vercel.com/changelog/dependencies-can-now-be-installed-with-a-custom-command) to make it available before the Build Step?
> 
> ![image](https://user-images.githubusercontent.com/8247105/181845332-36b3551c-89da-4a77-b16f-fcc313c6dbae.png)

Thank you for your reply though I've found another way to achieve the same goal. I'll try this later on! Have a nice day. @amyegan

**Comment by chienhsiang-hung** [comment]
I've used a similar module which is also powered by wkhtmltopdf on AWS. 
I've to set the bin path in order to use **pdfkit**: `pdfkit.configuration(wkhtmltopdf='/opt/bin/wkhtmltopdf')`

If you could tell me where would be the path that packages installed by **requirement.txt**, I think it could probably work. Thank you.
